src/main/kotlin/Release.kt

```diff
@@ -289,15 +303,20 @@ class Git(
         "git add ${files.filterNotNull().joinToString(separator = " ")}".execute()
     }
 
-    private fun createPullRequest(version: String, ticket: String) {
-        listOf(
+    private fun createPullRequest(version: String, ticket: String, labels: List<String>) {
+        mutableListOf(
             "hub", "pull-request",
             "-m", """Prepare next version $version
                     |
                     |Jira: [$ticket](https://ellation.atlassian.net/browse/$ticket)
                     |
                     |This PR was created [automatically](https://github.com/crunchyroll/android-library-release-script).""".trimMargin()
-        ).execute()
+        ).apply {
+            if (labels.isNotEmpty()) {
+                add("-l")
+                add(labels.joinToString(separator = " "))
```

Comment:
```suggestion
                add(labels.joinToString(separator = ","))
```

shouldn't labels be separated by coma?  🤔

Reply:
nope, this is noot how Clickt parses the multiple arguments

Reply:
Let me check it out 👍

Reply:
Plz have a look [here](https://github.com/crunchyroll/etp-android-app/blob/a81235f674401a438b5ec7db28f615b0256f2112/kscripts/src/main/kotlin/automate/hotfix/HotfixPr.kt#L118)

Reply:
Yeah, indeed

![image](https://user-images.githubusercontent.com/37998373/146210352-e528613e-8e2b-4602-b04d-0bef1211ecb4.png)